analyzers/src/SonarAnalyzer.CFG/ShimLayer/AnalysisContext/SymbolStartAnalysisContextWrapper.cs

```diff
@@ -135,10 +128,11 @@ static SymbolStartAnalysisContextWrapper()
             var cast = (Action<CodeBlockStartAnalysisContext<CS.SyntaxKind>>)action;
             RegisterCodeBlockStartActionCS(RoslynSymbolStartAnalysisContext, cast);
         }
-        else if (languageKindType == typeof(VB.SyntaxKind))
+        else if (languageKindType.FullName == "Microsoft.CodeAnalysis.VisualBasic.SyntaxKind")
         {
-            var cast = (Action<CodeBlockStartAnalysisContext<VB.SyntaxKind>>)action;
-            RegisterCodeBlockStartActionVB(RoslynSymbolStartAnalysisContext, cast);
+            // See https://github.com/SonarSource/sonar-dotnet/pull/9028 for how to implement this
+            // Attention: Do not add a package reference to Microsoft.CodeAnalysis.VisualBasic.Workspaces.
+            // It creates hard to detect file not found errors in .NET SDK 3 and .NET SDK 5
```

Comment:
The reason is not .NET SDK 3 and 5 - they're not supported.

But they add complexity in some scenarios which we cannot fully understand...